Artifacts/windows-azuredevops-multiple-agents/artifactfile.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "Azure Pipelines Agent(s)",
+    "publisher": "Utkarsh Shigihalli",
+    "description": "Downloads latest Azure Pipelines agent for the account, and installs the specified number instances of the agents inside one VM. Registers the agent with the Azure DevOps account, and adds agents to the specified agent pool. Note - If the specified agent folder already exists it will be overwritten.",
```

Comment:
Consider making the overwrite of existing agents an option, since it is destructive. A user may or may intend to overwrite an existing agent. So, if the `$Overwrite` flag is set, then you can do it. Make sure that it's default is `False`. Then check for this condition inside your script and fail accordingly. Better make the user aware.